ci/scripts/gitlab/report_test_results.py

```
@@ -181,21 +256,28 @@ def main():
-        junit_data = parse_junit(args.junit_file)
-        coverage_data = parse_coverage(args.coverage_file)
-
-        report_messages = build_messages(junit_data, coverage_data)
+        if has_health_arg:
+            health_data = parse_model_health(args.model_health_json)
+            report_messages = build_model_health_messages(health_data)
+        else:
+            junit_data = parse_junit(args.junit_file)
+            coverage_data = parse_coverage(args.coverage_file)
+            report_messages = build_messages(junit_data, coverage_data)
     except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)